### PR TITLE
db new documentation

### PR DESCRIPTION
**Pull request by mdallago**

### CODE REVIEW

File: content/user_guide/installation.md [hunk 1]
```diff
@@ -28,6 +28,56 @@ conan install bitprim-node-exe/0.X@bitprim/stable -o currency=BTC
...
 ```
+### Database selection
+
+Bitprim includes three databases modes, adapted to distinct uses cases.
+
+- pruned
+- default
+- full
+
+The dabatase mode selection is via the *db* setting in the conan command line.
```
Comment (dario-ramos) [inline content/user_guide/installation.md]: is done via*

File: content/user_guide/installation.md [hunk 2]
```diff
@@ -28,6 +28,56 @@ conan install bitprim-node-exe/0.X@bitprim/stable -o currency=BTC
...
+
+#### Pruned
+
+In this mode, the node only has the block's headers and the UTXO. This mode is optimized for miners and nodes that only requires validates new blocks and follow the chain.
```
Comment (dario-ramos) [inline content/user_guide/installation.md]: block headers*

Comment (dario-ramos) [inline content/user_guide/installation.md]: only require validating new blocks and following the chain*

File: content/user_guide/installation.md [hunk 3]
```diff
@@ -28,6 +28,56 @@ conan install bitprim-node-exe/0.X@bitprim/stable -o currency=BTC
...
+conan install bitprim-node-exe/0.X@bitprim/stable -o db=default
+
+or directly
+
+conan install bitprim-node-exe/0.X@bitprim/stable
+```
+
+#### Full
+
+This is the most complete mode. In this mode, the node store the full blocks, UTXO and maintains various indexes to speed up some queries.
```
Comment (dario-ramos) [inline content/user_guide/installation.md]: the node stores* the full blocks

File: content/user_guide/installation.md [hunk 4]
```diff
@@ -28,6 +28,56 @@ conan install bitprim-node-exe/0.X@bitprim/stable -o currency=BTC
...
+
+- Transactions by hash
+- Full history by address
+- Spends outputs
```
Comment (dario-ramos) [inline content/user_guide/installation.md]: Spent* outputs

File: content/user_guide/installation.md [hunk 5]
```diff
@@ -28,6 +28,56 @@ conan install bitprim-node-exe/0.X@bitprim/stable -o currency=BTC
...
+
+#### Full
+
+This is the most complete mode. In this mode, the node store the full blocks, the UTXO and maintains various indexes to speed up some queries.
+
+- Transactions by hash
+- Full history by address
+- Spends outputs
+
+This is the best mode for an Insight API node or if you want to use RPC to query the node.
```
Comment (fpelliccioni) [inline content/user_guide/installation.md]: Clarify that RPC works in all the modes, responding whatever available...

Comment (mdallago) [reply content/user_guide/installation.md]: RPC queries works in all modes, but not all request works in all modes. ?

**Pull request merged**